bundles/statistics/statsgrid2016/components/IndicatorParameters.js

```diff
@@ -87,14 +86,24 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParameters', function
             var placeholderText = (panelLoc.selectionValues[selected] && panelLoc.selectionValues[selected].placeholder) ? panelLoc.selectionValues[selected].placeholder : panelLoc.defaultPlaceholder;
             var label = (locale.parameters[selected]) ? locale.parameters[selected] : selected.id;
             var tempSelect = jQuery(me.__templates.select({id: selected, label: label}));
-            var options = {
-                placeholder_text: placeholderText,
-                allow_single_deselect: true,
-                disable_search_threshold: 10,
-                width: '100%'
-            };
+            var options;
+            if (selections[selected].time) {
+                options = {
+                    placeholder_text: placeholderText,
+                    multi: true
+                };
+                // assign the time describing property as "time" in the object
+                var old = selected;
+                selected = 'time';
+                Object.defineProperty(selections, selected, Object.getOwnPropertyDescriptor(selections, old));
+                delete selections[old];
```

Comment:
So you are renaming the variable to "time"? Doesn't this change what the getValues() return? If it does it will break the actual data fetching as it will try to ask for something like "time=2018"  from the statistical API which expects for example in SotkaNet to get "year=2018"

Reply:
In addition this causes problems if there's an existing variable named "time" for the indicator.